complete/03_ingress/app.ingress.yaml

```
@@ -0,0 +1,23 @@
+# You may need to change the ingress class below,
+# to match the Nginx Ingress configuration.
+#
+# Access this INgress by specifying the HOST header.
+# Replace ${load_balancer_ip} with the IP of an Ingress Controller LB.
+# curl --header 'HOST: test.domain.com' ${load_balancer_ip}/
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: webapp
+  namespace: k8s-workshop
+  annotations:
+    kubernetes.io/ingress.class: nginx
+spec:
+  rules:
+  - host: test.domain.com
+    http:
+      paths:
+      - path: /
+        backend:
+          serviceName: webapp
+          servicePort: 80
```

```suggestion
          service:
            name: webapp
            port: 80
```

This could be related to changing the apiVersion, but this is the format used in documentation and when looking at the godocs for ingress. https://kubernetes.io/docs/concepts/services-networking/ingress/#the-ingress-resource